src/mds/FSMap.cc

```diff
@@ -687,6 +696,8 @@ void FSMap::promote(
     // An existing rank is being assigned to a replacement
     info.state = MDSMap::STATE_REPLAY;
     mds_map.failed.erase(assigned_rank);
+    if (mds_map.standby_count_wanted == 0)
```

Comment:
It's not obvious to me why this belongs here: it seems like we're only going to miss setting standby_count_wanted initially on a healthy system with one active and one standby -- if the standby dies we won't warn, unless the active has died and been replaced previously.